ghost/i18n/locales/tr/portal.json

```diff
@@ -25,12 +25,12 @@
     "Cancel subscription": "Aboneliği iptal et",
     "Cancellation reason": "İptal sebebi",
     "Change": "Değiştir",
-    "Change plan": "",
+    "Change plan": "Plan Değiştir",
     "Check spam & promotions folders": "Spam ve promosyonlar klasörlerini kontrol edin",
     "Check with your mail provider": "Posta sağlayıcınızla kontrol edin",
     "Choose": "Seç",
     "Choose a different plan": "Farklı bir plan seç",
-    "Choose a plan": "",
+    "Choose a plan": "Bir Plan Seçin",
```

Comment:
Is this (lack of) spacing and the capitalization correct?

Reply:
Sorry - spacing looks perfectly fine here, not sure what was up on my other screen.  The question about capitalization remains.

Reply:
Thank you for your notification, I have made the correction.